src/server/scripts/Northrend/Naxxramas/boss_sapphiron.cpp

```diff
@@ -303,6 +306,7 @@ class boss_sapphiron : public CreatureScript
                             case EVENT_LIFTOFF:
                                 Talk(EMOTE_AIR_PHASE);
                                 me->SetDisableGravity(true);
+                                me->SetHover(true);
```

Comment:
Browse through [3.3.5/Ulduar] Miniron's Hover Mode (#15177),  [3.3.5/Ulduar] XT002 Entering the ground (#15032) and  [3.3.5] VMap data is sometimes not correct (#15161) for ideas.